### ISSUE TITLE
Accessibility: Missing labels on fundraising heart SVG

### ISSUE THREAD

**Issue by Mr-Sunglasses** [django/djangoproject.com#2552]
The fundraising page has accessibility violations  of the [WCAG "Text Labels and Names" criterion (Content with images must be labeled).](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Text_labels_and_names?utm_source=devtools&utm_medium=a11y-panel-checks-text-label&utm_campaign=default#Content_with_images_must_be_labeled) for the heart svg showing the funding goal.

<img width="1728" height="1117" alt="WCAG 'Text Labels and Names' criterion (Content with images must be labeled) error in heart svg showing the funding goal" src="https://github.com/user-attachments/assets/0d1b4a3f-1acd-4bf2-ad72-a59b9a8462da" />
